src/rootStyles.tsx

```diff
@@ -10,7 +10,7 @@ export const colors = {
     LIGHT_GREY: "#cfd0d0",
     DARK_BLUE_GREY: "#3b4856",
     logoLightBlue: "#0B9FEC",
-    logoDarkBlue: "#1C65BE"
+    logoDarkBlue: "#1E66BE"
```

Comment:
this one originated as a typo, I believe

Reply:
Ah, nice catch - it actually originated from before I knew there was an SVG version of the logo where we can get the true hexcodes for the logo colors (the above was generated using a program for sampling colors from images). I think `logoLightBlue` might be slightly off as well.

Reply:
yeah, it's #0C9FEB